frontend/lib/Repository/EndpointRepository/EndpointRestRepository.dart

```diff
@@ -1,54 +1,55 @@
-import 'package:adpv_frontend/Repository/AbstractEndpointRepository.dart';
+import 'package:adpv_frontend/DataModels/EndpointSummary.dart';
 import 'package:dio/dio.dart';
-import '../Common/Consts.dart';
-import '../Common/URLs.dart';
-import '../DataModels/EnableField.dart';
-import '../DataModels/EndpointData.dart';
-import '../DataModels/EndpointSummary.dart';
 
-class BackendResponse<T>{
+import '../../Common/Consts.dart';
+import '../../Common/URLs.dart';
+import '../../DataModels/EnableField.dart';
+import '../../DataModels/EndpointData.dart';
+
+class BackendResponse<T> {
   final T data;
   final String error;
 
   BackendResponse(this.data, this.error);
 
-  BackendResponse.fromJson(Map json) :
-        data = json["data"],
+  BackendResponse.fromJson(Map json)
+      : data = json["data"],
         error = json["error"];
 }
 
-class RestEnpointRepository implements AbstractEndpointRepository {
-  final Dio client;
+class EndpointRestRepository {
+  final Dio client = Dio();
 
-  RestEnpointRepository(this.client);
+  EndpointRestRepository();
 
-  @override
-  Future<List<EndpointSummary>> getEndpointSummary() async {
+  Future<List<EndpointSummary>> getEndpointSummaryList() async {
+    await Future.delayed(const Duration(seconds: 1)); //for my dear reviewer :)
```

Comment:
Potrzebujemy naprawdę tych sztucznych spowalniaczy?

Reply:
nie potrzebujesz to usuń, są imo bardzo pomocne przy pracy z Future<>